source/Calamari/Deployment/PackageRetention/PackageJournalCommandDecorator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Calamari.Commands.Support;
+using Calamari.Common.Plumbing.Deployment.PackageRetention;
+using Calamari.Common.Plumbing.Extensions;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+
+namespace Calamari.Deployment.PackageRetention
+{
+    public class PackageJournalCommandDecorator : ICommandWithArgs
+    {
+        readonly ILog log;
+        readonly ICommandWithArgs command;
+        readonly IManagePackageUse journal;
+        readonly bool retentionEnabled = false;
+
+        PackageIdentity Package { get; }
+        ServerTaskId DeploymentTaskID { get; }
```

Comment:
And here

Reply:
They're everywhere!
Thanks for spotting those!